in-adapter-api/src/test/java/gohigher/auth/support/JwtProviderTest.java

```diff
@@ -0,0 +1,140 @@
+package gohigher.auth.support;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Date;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import gohigher.global.exception.GoHigherException;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+
+@DisplayName("JwtProvider 클래스의")
+class JwtProviderTest {
+
+	private static final long ACCESS_TOKEN_EXPIRE_LENGTH = 100000;
+	private static final int REFRESH_TOKEN_EXPIRE_LENGTH = 300000;
+	private static final int UNIT_TO_CONVERT_MILLI_TO_SECOND = 1000;
+	private static final String SECRET = "secretKey".repeat(6);
+
+	private final JwtProvider jwtProvider = new JwtProvider(SECRET, ACCESS_TOKEN_EXPIRE_LENGTH,
+		REFRESH_TOKEN_EXPIRE_LENGTH);
+
+	@DisplayName("createToken 메서드는")
+	@Nested
+	class createToken {
+
+		@DisplayName("value가 정해지면")
+		@Nested
+		class Value {
+			Long userId = 1L;
+			Date now = new Date();
+
+			@DisplayName("엑세스 토큰이 정상적으로 만든다")
```

Comment:
```suggestion
			@DisplayName("엑세스 토큰이 정상적으로 만든다.")
```
온점을 붙이는게 좋을까요, 없애는게  좋을까요? 저는 없는 편을 선호하지만 전에 예시에 있던 사진에는 있어서 붙였었습니다!